addons/mail/static/src/model/record_internal.js

```diff
@@ -68,6 +62,7 @@ export class RecordInternal {
     proxyUsed = new Map();
     /** @type {string} */
     localId;
+    gettingField = false;
```

Comment:
Is this a new improvement? Commit makes no mention of this change.
It feels like it should be separated from the rest of the diff.

Reply:
This is just a flag to by-pass reactive while reading a non-field value on record. Without this there are infinite loops in some cases